### DISCUSSION TITLE
Feature request: Always select wallet at application start, if there is only one. (Skip wallet selection screen at app start.)

### DISCUSSION THREAD

**Discussion by LeoniePhiline** [Emurgo/yoroi-mobile#1869] [Ideas]
I have only one wallet in Yoroi. 
Yet still each time I start the app, I need to select the wallet. 
As a user I want it automatically selected. 

![9503B74F-A7BE-4C75-9C05-393CC8855A64](https://user-images.githubusercontent.com/22329650/126073805-cd3f3f1e-f257-45d6-a821-730e5aa5fd5e.png)

**Comment by stackchain** [comment]
@vsubhuman we can discuss it with @Marta-Ryzhok and Sergio for the revamp.
